bin/node/cli/src/chain_spec.rs

```diff
@@ -511,16 +511,19 @@ fn pangolin_development_genesis() -> pangolin_runtime::GenesisConfig {
 		pallet_im_online: Default::default(),
 		pallet_authority_discovery: Default::default(),
 		darwinia_democracy: Default::default(),
-		pallet_collective_Instance1: pangolin_runtime::CouncilConfig {
-			phantom: PhantomData::<pangolin_runtime::CouncilCollective>,
-			members: collective_members.clone(),
```

Comment:
Do not need this genesis config now? Give more information to help with the review.